### PR TITLE
Fixing oracle jdk 8 url and updating to 8u202.

### PR DESCRIPTION
**Pull request by dezka**
### Description

Updating the Oracle JDK 8 url and checksum again.

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: resources/jce.rb [hunk 1]
```diff
@@ -49,7 +49,8 @@
   jre_path = node['java']['install_type'] == 'jdk' ? 'jre' : ''
 
   if node['os'] == 'windows'
-    include_recipe 'windows'
```
Comment (tas50) [inline resources/jce.rb]: This can actually go away entirely. The windows cookbook has nothing in the default recipe

**Comment by tas50** [comment]
Looks good other than that old windows bit that can just fade off into the sunset. Thanks for putting this together. I meant to update it yesterday and then something distracted me as usual

File: resources/jce.rb (hunk 1 above)
Comment (dezka) [reply resources/jce.rb]: Gotcha, yeah @majormoses and I weren't sure on if it was actually needed. Thanks

**Comment by majormoses** [comment]
I think the cookbook is still needed per https://circleci.com/gh/sous-chefs/java/199?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link and https://github.com/chef-cookbooks/windows/blob/master/resources/zipfile.rb

**Comment by tas50** [comment]
Yeah I need to yank the entire bit that uses the windows zip, but this PR isn't the cause of that failure.

**Comment by majormoses** [comment]
Due to some of the breaking changes merged into master but not released I backported the change into the 3.x codebase and released this as: https://supermarket.chef.io/cookbooks/java/versions/3.2.1

**Pull request merged**